mentoring/groupmentoring.md

```diff
@@ -0,0 +1,34 @@
+# Group Mentoring for Contributor Ladder Growth
+*This is a work in progress*
+
+## Summary
+This is an experimental group mentoring program where individuals will work self-paced in a semi-structured learning environment over the course of three months. Want to get `/lgtm` rights? Be a decision maker in an OWNERS file? This could be a great way to get the knowledge you need, some accountability to do it.  Through this program you will get the opportunity to interact with active contributors who may be on the same path as you as well  very experienced Kubernetes contributors (Pilots). 
+
+## How This Works
+This program is built around our [community membership guidelines](https://github.com/kubernetes/community/blob/master/community-membership.md). This doc shows how you can grow through the project in different roles and provides the requirements to get to each rung on our ladder.
```

Comment:
As we are inside the same repository we can use relative links instead: [community membership guidelines](/community-membership.md)

Reply:
As a non native speaker it might make it easier to use "level/stage" instead of "rung". Feels more complicated to understand that way and it might make it harder for people without a good english level.

Reply:
Thank you for the suggestion! I appreciate that! I made all of your suggested changes.